### ISSUE TITLE
Events Audit Trail: Log real client IP as well

### ISSUE THREAD

**Issue by akanouras** [zitadel/zitadel#9106]
### Preflight Checklist

- [x] I could not find a solution in the existing issues, docs, nor discussions
- [x] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

I'm running Zitadel behind a reverse proxy and because of that I cannot see the clients' real IP address in the audit trail.

An example of a "Password check succeeded" event:
```json
{
  "acceptLanguage": "en-US,en;q=0.8,el-GR;q=0.5,el;q=0.3",
  "id": "299392133956154553",
  "remoteIP": "192.168.160.124",
  "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:133.0) Gecko/20100101 Firefox/133.0",
  "userAgentID": "299392133452838073"
}
```

### Describe your ideal solution

Ideally, the above event would look like this:

```json
{
  "acceptLanguage": "en-US,en;q=0.8,el-GR;q=0.5,el;q=0.3",
  "id": "299392133956154553",
  "remoteIP": "192.168.160.124",
  "X-Forwarded-For": "100.100.100.100",
  "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:133.0) Gecko/20100101 Firefox/133.0",
  "userAgentID": "299392133452838073"
}
```

Basically a dump of the headers Forwarded, X-Forwarded-For, X-Real-IP, if any of them exist.

### Version

v2.54.8

### Environment

Self-hosted

**Comment by muhlemmer** [comment]
We found a bug where we actually look for `X-Forwared-For` and we didn't use a canonical lookup:

https://github.com/zitadel/zitadel/blob/42cc6dce79bb0659956d358d142a5ecf2cac59f3/internal/api/http/header.go#L110-L119

A fix should be available soon
